src/TimeSteppers/runge_kutta_3.jl

```diff
@@ -110,14 +107,13 @@ function time_step!(model::AbstractModel{<:RungeKutta3TimeStepper}, Δt; callbac
 
     tick!(model.clock, first_stage_Δt; stage=true)
     store_tendencies!(model)
-    update_state!(model, callbacks)
     update_particle_properties!(model, first_stage_Δt)
 
     #
     # Second stage
     #
 
-    calculate_tendencies!(model, callbacks)
+    update_state!(model, callbacks)
```

Comment:
We want to leave `update_state!` where it is right?

Reply:
it is there, just changed for a hot second 😅